src/constant.c

```diff
@@ -31,10 +31,6 @@ const int AGE_TYPE_MAP[N_AGE_GROUPS] = {
 	AGE_TYPE_CHILD,
 	AGE_TYPE_CHILD,
 	AGE_TYPE_ADULT,
-	AGE_TYPE_ADULT,
-	AGE_TYPE_ADULT,
```

Comment:
Removing these 4 AGE_TYPE_ADULT here is causing the failing test. 
AGE_TYPE_MAP is used to for assigning the age_type which is then used by the individual to see which parameters to use for calculating the number of random interactions.

Reply:
This fixes my tests, thanks!